airflow/dags/gtfs_views/gtfs_schedule_fact_daily_feed_files.sql

```diff
@@ -35,16 +35,25 @@ raw_daily_files AS (
         , T1.is_agency_changed
         , T1.full_path
         , FALSE AS is_interpolated
+        , T3.parse_error_encountered
 
         -- calculate the leading date, so we can fill in missing rows, where
         -- extraction failed to run.
         , LEAD(T1.calitp_extracted_at)
-            OVER (PARTITION BY calitp_itp_id, calitp_url_number ORDER BY T1.calitp_extracted_at)
+            OVER (PARTITION BY T1.calitp_itp_id, T1.calitp_url_number ORDER BY T1.calitp_extracted_at)
             AS tmp_next_date
 
     FROM `gtfs_schedule_history.calitp_files_updates` T1
     JOIN `views.gtfs_schedule_dim_feeds` T2
-        USING (calitp_itp_id, calitp_url_number)
```

Comment:
Can we put this using clause back in? This seems like the construct it was designed for (joining on keys of the same name)

Reply:
I thought that would lead to an ambiguous error due to the addition of the `T3` join, but looks like it's ok on the first join. I added it back in 4b37dc1.